app/dashboard/views.py

```diff
@@ -555,6 +555,23 @@ def unrated_bounties(request):
         'unrated': unrated_count,
     }, status=200)
 
+@csrf_exempt
+@require_POST
+def remove_portfolio(request):
+    portfolio_id = int(request.POST.get('portfolio_id'))
+    handle = request.user.profile.handle
+
+    try:
+        profile = profile_helper(handle, True)
+        portfolio_item = PortfolioItem.objects.get(id = portfolio_id)
+        portfolio_item.delete()
```

Comment:
Instead of deleting, lets set a property on the PortfolioItem such that if the property is True, it doesn't get displayed. Deleting data is generally something we should do manually as part of archival processes. Interesting discussion here: https://stackoverflow.com/questions/2549839/are-soft-deletes-a-good-idea